Frends.Utilities.AlwaysTrue/Frends.Utilities.AlwaysTrue/AlwaysTrue.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Frends.Utilities.AlwaysTrue;
+
+/// <summary>
+/// Utilities AlwaysTrue task.
+/// </summary>
+public class Utilities
+{
+    /// <summary>
+    /// Return boolean value true.
+    /// [Documentation](https://tasks.frends.com/tasks/frends-tasks/Frends.Utilities.AlwaysTrue)
+    /// </summary>
+    /// <returns>bool True</returns>
+    public static bool AlwaysTrue()
```

Comment:
Tasks should always return a result-object, so please add a result object which is returned from this task.